scripts/test-installers.py

```
@@ -0,0 +1,304 @@
+POSIX_PLATFORMS = (
+    ("linux", "x86_64"),
+    ("linux", "aarch64"),
+    ("macos", "x86_64"),
+    ("macos", "aarch64"),
+)
+
+
+class QuietHandler(http.server.SimpleHTTPRequestHandler):
+    """Serves fixture releases without writing request logs."""
+
+    def log_message(self, format: str, *args: object) -> None:
```

Ruff (0.15.21)

[error] Function argument `format` is shadowing a Python builtin

(A002)